module-bsp/bsp/common.hpp

```diff
@@ -15,13 +15,13 @@ namespace bsp
     /// Only a few thresholds are available in the current configuration
     enum class CpuFrequencyHz
```

Comment:
I will change name to MHz after approval (it touches a lot of files so do not want to introduce it now to clear up review).

Reply:
Wouldn't it be just easier to name it like:
`
level_4MHZ,
level_12MHz,
...
` ?
Probably the values are not use anywhere

Reply:
Well they are used now to print in that new window :D